package.json

```diff
@@ -40,11 +40,14 @@
     "enzyme-to-json": "^3.0.1",
     "eslint": "^4.7.2",
     "eslint-plugin-react": "^7.4.0",
+    "jest": "^22.1.4",
     "node-sass-chokidar": "0.0.3",
     "nodemon": "^1.12.1",
     "react-scripts": "^1.0.13",
     "react-test-renderer": "^16.0.0",
-    "sass-lint": "^1.12.1"
+    "sass-lint": "^1.12.1",
+    "superagent": "^3.8.2",
```

Comment:
Do we need to explicitly require superagent in the package.json?  I'm thinking that since supertest depends on superagent, supertest might install it as a dependency automatically.

Reply:
It would, but I thought it was best practice to include exactly what you need and use as best practice. Also, I have first hand experience with NPM pruning things that you don't explicitly require (which is ridiculous) -- one of the many examples: https://github.com/npm/npm/issues/17929